mingw-w64-glad/PKGBUILD

```
@@ -0,0 +1,40 @@
+# Maintainer: kasper93 <kasper93@gmail.com>
+
+_realname=glad
+pkgbase=mingw-w64-python-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-python-${_realname}"
+pkgver=2.0.4
+pkgrel=1
+pkgdesc='Multi-Language Vulkan/GL/GLES/EGL/GLX/WGL Loader-Generator based on the official specs (mingw-w64)'
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
+url='https://github.com/Dav1dde/glad'
+license=('MIT')
+depends=("${MINGW_PACKAGE_PREFIX}-python"
+         "${MINGW_PACKAGE_PREFIX}-python-jinja"
+         "${MINGW_PACKAGE_PREFIX}-python-lxml"
+         "${MINGW_PACKAGE_PREFIX}-python-setuptools")
+makedepends=("${MINGW_PACKAGE_PREFIX}-python-build"
+             "${MINGW_PACKAGE_PREFIX}-python-installer"
+             "${MINGW_PACKAGE_PREFIX}-python-wheel")
```

```suggestion
         "${MINGW_PACKAGE_PREFIX}-python-jinja")
makedepends=("${MINGW_PACKAGE_PREFIX}-python-build"
             "${MINGW_PACKAGE_PREFIX}-python-installer"
             "${MINGW_PACKAGE_PREFIX}-python-setuptools"
             "${MINGW_PACKAGE_PREFIX}-python-wheel")
```
Only jinja is required https://github.com/Dav1dde/glad/blob/v2.0.4/setup.py#L34 https://github.com/Dav1dde/glad/blob/v2.0.4/requirements.txt